docs/DeveloperGuide.md

```diff
@@ -346,7 +347,29 @@ is used to handle any valid date time values. Each of this meeting are stored in
 contains each of the meetings related to each other stored in an ```ObservableList```. The ``` MeetingManager ``` is
 used to manage any operations that require viewing or sorting of meetings from the ```MeetingList``` class.
 
-#### What designs were considered:
+The operations for adding and deleting meeting are handled by `AddMeetingCommand` and `DeleteMeetingCommand`, which are supported by `AddMeetingCommandParser` and `DeleteMeetingCommandParser` respectively.
+
+1. The user enters `meeting-add 2 d/Meet for practical exam s/20/04/2024 15:00` to add a meeting or `meeting-delete 1 i/1` to delete a meeting.
+2. `Logic Manager` receives the user input which is parsed by `StaffConnectParser`.
+3. After splitting the user input into `commandWord` and `arguments` based on the regex pattern of the user input, the `StaffConnectParser` invokes the `AddMeetingCommandParser`or`DeleteMeetingCommandParser` based on the `commandWord`. Calling the method `parse` with `arguments` as the method arguments, and getting supported by parsing methods from `ParsedUtil`. 
+4. `AddMeetingCommand` or `DeleteMeetingCommand` is created with the parsed values.
+5. `Logic Manager` executes the `AddMeetingCommand` or `DeleteMeetingCommand`, which handles adding/removing meeting from the `Person` respectively and updates the model with the new information.
+
+Below is the sequence diagram for parsing inputs with  `AddMeetingCommandParser` executing `meeting-add 2 d/Meet for practical exam s/20/04/2024 15:00`:
```

Comment:
I realize that the image for AddMeetingParserSequenceDiagram does not match with this description here. The description states that the sequence diagram below is executing `meeting-add 2 d/Meet for practical exam s/20/04/2024 15:00` however the image shows the variable name instead. 

Perhaps the image of the diagram should reflect the given description to reduce confusion instead?

Reply:
Are you referring to `userInput` and `argument` in the `AddMeetingParserSequenceDiagram`? The values are described in the note heading in lines 359-361. Also I opted to use `userInput` and `arguments` placeholder because putting the the actual values in the diagram would make the diagram too small to be seen

Reply:
Is there a reason why you've opted to that method instead of breaking your diagram further by using a ref frame?

Reply:
Actually I am confused, the method interactions (calls ) are not omitted, but I just used placeholder(representative) text values in the method parameters to indicate their purpose. The values are stated in the note header. Also correct me if I am wrong but isn't ref frame used when used when omitting interactions from the main sequence diagram  instead (functions calls drawn in a separate diagram )

Reply:
Nvm I think to make things simpler I just standardise with the same layout as yours in #268

Reply:
> Actually I am confused, the method interactions (calls ) are not omitted, but I just used placeholder(representative) text values in the method parameters to indicate their purpose. The values are stated in the note header. Also correct me if I am wrong but isn't ref frame used when used when omitting interactions from the main sequence diagram instead (functions calls drawn in a separate diagram )

I apologize for the confusion caused. I'm inquiring about what's the more appropriate way of doing the diagram and documentation. 

Regarding ref frame, you are correct that it is used when omitting interactions from the main sequence diagram. However, the reason for omitting interactions also includes breaking complicated sequence diagrams into smaller fragments.
Quoted from the [textbook](https://nus-cs2103-ay2324s2.github.io/website/se-book-adapted/chapters/uml.html#reference-frames):
> UML uses ref frame to allow a segment of the interaction to be omitted and shown as a separate sequence diagram. Reference frames help you to **break complicated sequence diagrams into multiple parts** or simply to omit details you are not interested in showing.

Reply:
I think both ways are fine, but to make things standard i just follow your layout

Reply:
Has been fixed in the latest commit!